SCons/__init__.py

```diff
@@ -1,9 +1,9 @@
 __version__="4.1.1a"
 __copyright__="Copyright (c) 2001 - 2021 The SCons Foundation"
-__developer__="bdbaddog"
-__date__="2021-02-26 22:38:25"
-__buildsys__="ProDog2020"
-__revision__="3fa309f8fc5472fa067d7c04481fd0a5d3e86352"
-__build__="3fa309f8fc5472fa067d7c04481fd0a5d3e86352"
+__developer__="millsd"
```

Comment:
Don't include this file in your commit. It's autogenerated on every full or doc build.